code/game/objects/items/devices/remotecyborg.dm

```diff
@@ -0,0 +1,296 @@
+* Cyborg Control Boards: To use, you install this instead of a MMI when making a robot. It will create a cyborg as normal, but without a player to control it.
+* With the syndicate controller, you can control the cyborg like any other player would, selecting a module, able to change your name, communicate with the AI.
+* At the same time, you are linked to the cyborg consoles. So they can lock you down, and self-destruct the robot
+* Syndicate Controller: Imagine a controller with a screen and a keyboard, you can see the borgs you put the cameras on, or control your own borg.
+*
+* Other things:
+* Syndieborg: Just a borg that has the Cyborg Control Board. I made this so I could overwrite the login message, it doesn't make sense to have the player recieve the
+* law messages any time he connects to the borg. So it now displays a message about how to log off from the borg.
+* Frequency: Every object in this list has a frequency. You can make your own frequency by editing the variables. They all default to 'syndicate'.
+* What it means is, only boards on the syndicate frequency can communicate with syndicate controllers. You can have your own 'network', if you could say so.
+* Active: Just a 0 or 1 flag to state if the object (Camera or Control board) is online. Setting it to 0 will not instantly disconnect the player from the camera or borg though.
+* It is just a flag for when making the list. It updates when the mob with the camera, or board, dies.
+* Logoff Procedure: A small procedure made for the syndieborgs. It disconnects the player from the borg, allowing them to simply come back to their original body. Nice and easy.
+* 'Fuck Up Procs': It's the verifiers when the user, who is controlling or watching the camera, gets fucked up
+* (controller removed from his hands, downed, pushed, stunned, monkified, or anything that can disable him). It stops the user from viewing the camera,
+* or controlling the borg as soon as the code allowed me to. You still get a small grace moment (around a second) when dealing with the control boards because the process() method
+* has a timeout before it is called again.
+* Global Lists: Two lists, one for cameras, other for boards. Easy to understand. They just contain all active cameras/boards. When a borg dies, it is removed from there.
+* Attack Message Supression: Just overwritten methods to remove the "YOU FUCKING ATTACK THE TARGET WITH YOUR SECRET CAMERA".
+*
+* Stuff in other .DMs (If removing this module, please, mind these notes)
+* Added in robot.dm, attackby():
+* A verification about the camera, so it will not cause sparks
+* uplink_item.dm:
+* Added the box of items on the uplink_item.dm so the roboticist can pick the bundle to build a remote cyborg
+*
+*/
+
+var/list/rc_cameras = list()
+var/list/rc_control_boards = list()
+
+/*
+    START REMOTE CYBORG CAMERA SECTION
+*/
+/obj/item/device/syndicate_remote_cyborg_camera
```

Comment:
This is outright useless, given that the Security Camera computer can do the exact same thing.